packages/noya-state/src/selectors.ts

```diff
@@ -125,6 +129,27 @@ export const getSelectedLayersWithFixedRadius = (
   ).filter((layer): layer is Sketch.Rectangle => layer._class === 'rectangle');
 };
 
+export const getSelectedSwatches = (
```

Comment:
If this isn't used we should remove it, cause I'm not sure if there are other kinds of swatches besides colors. I think the other types are in a different place in the file format

Reply:
Deleted the extra selector